src/sentry/static/sentry/app/components/group/times.tsx

```diff
@@ -24,6 +24,8 @@ const Times = ({lastSeen, firstSeen}: Props) => (
     <div
       css={css`
         ${overflowEllipsis}
+        display: flex;
+        align-items: center;
```

Comment:
This aligns the clock icon with the text in Firefox.

Reply:
Could you add this comment inside the code?

IMO it'll be helpful so someone fiddling around on Chrome don't get rid of it by mistaking it as redundant.